### PR TITLE
Create Run.bat for EasyEDA

### PR DESCRIPTION
**Pull request by scarrrr316**
dont need to open cmd , just double click this batch file and drag the EasyEDA source file

### CODE REVIEW

File: InteractiveHtmlBom/Run.bat [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@echo off
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                                  Thankyou For Using Generate InteractiveBom
+echo                                              Powered By Qu1ck
+echo                                Bat Version : Powered By Scarrrr0725
```
Comment (qu1ck) [inline InteractiveHtmlBom/Run.bat:8]: This is better:

Thank you for using InteractiveHtmlBom
https://github.com/openscopeproject/InteractiveHtmlBom
Bat file by Scarrrr0725

File: InteractiveHtmlBom/Run.bat [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+echo                                                                                                                                                                                    -
+echo --------------------------------------------------------------------------------------------------------------------
+echo --------------------------------------------------------------------------------------------------------------------
+
+::delete --show-dialog after frist sart up and setting
```
Comment (qu1ck) [inline InteractiveHtmlBom/Run.bat:13]: typo: start

File: InteractiveHtmlBom/Run.bat [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+set FilePath=%~dp0
+set pyFilePath=%FilePath%generate_interactive_bom.py
+
+:convert
+set /p pathofEDASourceFile=Please Drag the EasyEDA PCB source file here :
+
+echo  Converting. . . . . . . . . .
+python %pyFilePath% %pathofEDASourceFile% %option%
+echo %pyFilePath% %pathofEDASourceFile%%option%
```
Comment (qu1ck) [inline InteractiveHtmlBom/Run.bat:24]: There should be a space here before %option% same as on the line above.

File: InteractiveHtmlBom/Run.bat [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                              EDA Source File is Converted  to Bom Successfully ! ! ! !
+echo                                    Thankyou For Using Generate InteractiveBom
```
Comment (qu1ck) [inline InteractiveHtmlBom/Run.bat:30]: Lets not be overly excited :)

"EDA source file is converted to bom successfully!"

No need for thanking user again, we did that in the intro banner.

File: InteractiveHtmlBom/Run.bat [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+echo                                                                                                                                                                                    -
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+
+CHOICE /C YN /N /M "Do you Want to Convert Next File  ? [Y/N]"
```
Comment (qu1ck) [inline InteractiveHtmlBom/Run.bat:35]: "Do you want to convert another file? [Y/N]"

File: InteractiveHtmlBom/Run.bat (hunk 1 above)
Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:8]: ok....sorry for my english

File: InteractiveHtmlBom/Run.bat (hunk 3 above)
Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:24]: oh........I fogot to delete this line，it used to test my bat file

File: InteractiveHtmlBom/Run.bat (hunk 4 above)
Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:30]: haha ,sorry for my english, again

File: InteractiveHtmlBom/Run.bat (hunk 5 above)
Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:35]: sorry for my english，emmmmmmm

File: InteractiveHtmlBom/Run.bat (hunk 2 above)
Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:13]: what's your mean?

Comment (scarrrr316) [reply InteractiveHtmlBom/Run.bat:13]: oh my god, I typed  wrong

**Comment by qu1ck** [comment]
Thanks. Do you want to try to make it kicad compatible like I explained above?
If not that's fine, this is good enough to merge. I'll take a stab at kicad compatibility later.

**Comment by scarrrr316** [comment]
oh, I have never use kicad, so I dont know about that python

**Comment by scarrrr316** [comment]
can I use this command to convert kicad file？

**Comment by qu1ck** [comment]
> can I use this command to convert kicad file？
python %pyfilepath%generate_interactive_bom.py kicadfile

Yes, except it needs to be python packaged with kicad, i.e. something like this:
`C:\Program Files\Kicad\bin\python.exe ...`

I posted above the registry key that stores kicad install location.

>And I saw that there are some files created by venv，and the files were not complete，how can I compete it ? Make it run without python of kicad or local.

Not sure what you mean

**Comment by scarrrr316** [comment]
emmmm ，I mean use virtualenv to create virtual python environment

**Comment by qu1ck** [comment]
And what is the issue with virtualenv? What files are not complete?

**Pull request merged**